`--showConfig` is ignoring missing `extends` file

<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker. Even if you think you've found a *bug*, please read the FAQ first, especially the Common "Bugs" That Aren't Bugs section!

Please help us by doing the following steps before logging an issue:
  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
  * Read the FAQ: https://github.com/Microsoft/TypeScript/wiki/FAQ

Please fill in the *entire* template below.
-->

<!-- Please try to reproduce the issue with `typescript@next`. It may have already been fixed. -->
**TypeScript Version:**  3.2.0-dev.20181117

and local build with 608a8307accdfdf0f0ae8fb8b356adedee939d09

<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
**Search Terms:**
showConfig

**Code**

```tsconfig.json
{
  "extends": "./notExists.tsconfig.json",
  "compilerOptions": {
    "target": "esnext",
    "module": "commonjs",
    "strict": true
  }
}
```

**Expected behavior:**
show `error TS5058: The specified path does not exist: '/foobar/notExists.tsconfig.json'.` error message.

**Actual behavior:**
don't report any error and dump incompleteness config value.

**Playground Link:** None

**Related Issues:** #27353

I made a patch to this issue.
If this is a problem, I can submit this patch.
https://github.com/vvakame/TypeScript/commit/d6d9c9ec2053080f4ac632357a422e28e03f7391

--- a/src/tsc/tsc.ts
+++ b/src/tsc/tsc.ts
@@ -136,7 +136,7 @@ namespace ts {
                 if (configParseResult.errors.length !== 0) {
                     updateReportDiagnostic(configParseResult.options);
                     configParseResult.errors.forEach(reportDiagnostic);
-                    return sys.exit(ExitStatus.DiagnosticsPresent_OutputsGenerated);
+                    return sys.exit(ExitStatus.DiagnosticsPresent_OutputsSkipped);
                 }
                 // tslint:disable-next-line:no-null-keyword
                 sys.write(JSON.stringify(convertToTSConfig(configParseResult, configFileName, sys), null, 4) + sys.newLine);

--- a/src/tsc/tsc.ts
+++ b/src/tsc/tsc.ts
@@ -133,8 +133,13 @@ namespace ts {
         if (configFileName) {
             const configParseResult = parseConfigFileWithSystem(configFileName, commandLineOptions, sys, reportDiagnostic)!; // TODO: GH#18217
             if (commandLineOptions.showConfig) {
+                updateReportDiagnostic(configParseResult.options);
+                configParseResult.errors.forEach(reportDiagnostic);
                 // tslint:disable-next-line:no-null-keyword
                 sys.write(JSON.stringify(convertToTSConfig(configParseResult, configFileName, sys), null, 4) + sys.newLine);
+                if (configParseResult.errors.length !== 0) {
+                    return sys.exit(ExitStatus.DiagnosticsPresent_OutputsGenerated);
+                }
                 return sys.exit(ExitStatus.Success);
             }
             updateReportDiagnostic(configParseResult.options);

--- a/src/tsc/tsc.ts
+++ b/src/tsc/tsc.ts
@@ -133,13 +133,13 @@ namespace ts {
         if (configFileName) {
             const configParseResult = parseConfigFileWithSystem(configFileName, commandLineOptions, sys, reportDiagnostic)!; // TODO: GH#18217
             if (commandLineOptions.showConfig) {
-                updateReportDiagnostic(configParseResult.options);
-                configParseResult.errors.forEach(reportDiagnostic);
-                // tslint:disable-next-line:no-null-keyword
-                sys.write(JSON.stringify(convertToTSConfig(configParseResult, configFileName, sys), null, 4) + sys.newLine);
                 if (configParseResult.errors.length !== 0) {
+                    updateReportDiagnostic(configParseResult.options);
+                    configParseResult.errors.forEach(reportDiagnostic);
                     return sys.exit(ExitStatus.DiagnosticsPresent_OutputsGenerated);
                 }
+                // tslint:disable-next-line:no-null-keyword
+                sys.write(JSON.stringify(convertToTSConfig(configParseResult, configFileName, sys), null, 4) + sys.newLine);
                 return sys.exit(ExitStatus.Success);
             }
             updateReportDiagnostic(configParseResult.options);

